autoload/vital/__vital__/VS/Vim/Syntax/Markdown.vim

```diff
@@ -54,7 +54,11 @@ function! s:apply(...) abort
 
       let l:filetype_group = substitute(toupper(l:filetype), '\.', '_', 'g')
       if !has_key(b:___VS_Vim_Syntax_Markdown.filetypes, l:filetype_group)
-        call s:_execute('syntax include @%s syntax/%s.vim', l:filetype_group, l:filetype)
+        try
+          call s:_execute('syntax include @%s syntax/%s.vim', l:filetype_group, l:filetype)
+        catch
+          unsilent echomsg printf('Fail to apply "syntax/%s.vim". Add "syntax/%s.vim" to runtimepath for suppress message', l:filetype, l:filetype)
```

Comment:
Thank you for your contribution.

I recommend to use `let g: markdown_fenced_languages = ['ts=typescript']` for suppressing the error message.
So this message should explain it. What do you think?

Reply:
And I think if this `syntax include` was failed, the below `syntax region` shouldn't execute.
So I think this try-catch should be removed.

Reply:
Thank you for comment!
ah, I did't know that.

I agree about that.

```
unsilent echomsg printf('Fail to apply "syntax/%s.vim". Add "let g:markdown_fenced_languages = ["%s=$FILETYPE"]" to enable syntax', l:filetype, l:filetype)
```

move above message to 

```
    catch /.*/
      unsilent echomsg string({ 'exception': v:exception, 'throwpoint': v:throwpoint })
    endtry
```
this catch block.

What about this?

Reply:
I think your suggestion is the correct way!

Reply:
Thank you!
Updated. PTAL 🙏